blockchain/reactor.go

```diff
@@ -22,8 +22,7 @@ const (
 	// BlockchainChannel is a channel for blocks and status updates (`BlockStore` height)
 	BlockchainChannel = byte(0x40)
 
-	defaultChannelCapacity = 1000
```

Comment:
why not leave this here if we still using it ? bad naming ?

Reply:
i think so, yeah. it does not explain a) why it's default? b) why 1000 is a default? c) is it supposed to be used for all channels?